input/fsh/modules/biospecimen.fsh

```diff
@@ -112,38 +119,19 @@ Description: "FHIR Profile for NCPI Sample"
 * collection.collectedDateTime ^short = "The age at which this biospecimen was collected. Could be expressed with a term, an age, or an age range. (for ages use http://hl7.org/fhir/StructureDefinition/cqf-relativeDateTime)"
 * collection.quantity 0..1 /*Sample.Quantity*/
 * collection.quantity ^short = "The total quantity of the specimen"
```

Comment:
Can we add a line similar to below to document that we require collection if it is a collected sample (as opposed to a derived one). This will add that show up in that difference table which I think is what most people will look at when they are writing their transforms. 

* collection ^short = "Some descriptive text explaining when you must use a collection (maybe including the collection.method can equal unknown if it isn't known...etc. ). "